server/command/help.go

```diff
@@ -11,7 +11,10 @@ import (
 
 func (c *Command) help(parameters ...string) (string, bool, error) {
 	resp := ""
-	for _, cmd := range cmds {
+	for _, cmd := range commandsWhenAutoConnectIsFalse {
+		if c.Config.AutoConnectUsers && cmd.Trigger == "connect" {
```

Comment:
Is there a constant made for `connect`? If yes, you should use that.

Reply:
No, there is no constant